include/zephyr/bluetooth/mesh/brg_cfg_cli.h

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_CLI_H__
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_CLI_H__
+
+#include <zephyr/bluetooth/mesh.h>
+#include <zephyr/bluetooth/mesh/brg_cfg.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup bt_mesh_brg_cfg_cli Bridge Configuration Client Model
+ * @ingroup bt_mesh
+ * @{
+ *  @brief API for the Bluetooth Mesh Bridge Configuration Client model
+ */
+
+struct bt_mesh_brg_cfg_cli;
+
+/**
+ *  @brief Bridge Configuration Client model Composition Data entry.
+ *
+ *  @param _cli Pointer to a @ref bt_mesh_brg_cfg_cli instance.
+ */
+#define BT_MESH_MODEL_BRG_CFG_CLI(_cli)                                                            \
+	BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_BRG_CFG_CLI, _bt_mesh_brg_cfg_cli_op, NULL, _cli,        \
+			 &_bt_mesh_brg_cfg_cli_cb)
+
+/** Mesh Bridge Configuration Client Status messages callback */
+struct bt_mesh_brg_cfg_cli_cb {
+	/** @brief Optional callback for Subnet Bridge Status message.
+	 *
+	 *  Handles received Subnet Bridge Status messages from a Bridge
+	 *  Configuration Server.
+
+	 *  @param cli      Bridge Configuration Client context.
+	 *  @param addr     Address of the sender.
+	 *  @param status   Status received from the server.
+	 */
+	void (*subnet_bridge_status)(struct bt_mesh_brg_cfg_cli *cli, uint16_t addr,
```

Comment:
My suggestion to shorten callbacks naming using prefix. Something like: `sbr_status` etc

Reply:
Will keep the name from the Mesh Protocol spec.